tools/setup/emoji/emoji_setup_utils.py

```diff
@@ -242,3 +238,7 @@ def generate_emoji_catalog(emoji_data):
     for category in emoji_catalog:
         emoji_catalog[category].sort(key=lambda codepoint: sort_order[codepoint])
     return emoji_catalog
+
+def image_available_for(emoji):
```

Comment:
I'd call this function `emoji_is_universal`, and the argument `emoji_dict` (or something other than `emoji`).